hedera-node/hedera-mono-service/src/main/java/com/hedera/services/state/logic/AwareNodeDiligenceScreen.java

```diff
@@ -62,6 +56,10 @@ public AwareNodeDiligenceScreen(
 
     public boolean nodeIgnoredDueDiligence(final DuplicateClassification duplicity) {
         final var accessor = txnCtx.accessor();
+        if (accessor.hasConsequentialUnknownFields()) {
```

Comment:
Better to add a comment here, why unknown fields is a problem instead of ignoring them (for future references 🙂).